lib/Driver/ToolChains.cpp

```diff
@@ -1283,10 +1284,10 @@ toolchains::Darwin::constructInvocation(const LinkJobAction &job,
     Arguments.push_back(context.Args.MakeArgString(LibProfile));
   }
 
-  // FIXME: We probably shouldn't be adding an rpath here unless we know ahead
-  // of time the standard library won't be copied.
```

Comment:
Please preserve this FIXME in the new code as well—it's referencing the rest of SR-1967.